backend/inspirehep/disambiguation/tasks.py

```diff
@@ -64,3 +67,21 @@ def disambiguate_signatures(self, clusters):
             disambiguation_assigned_clusters.labels("2+").inc()
             LOGGER.debug("Received cluster with more than 1 author.")
     db.session.commit()
+
+
+@shared_task(ignore_result=False, bind=True)
+def disambiguate_authors(self, record_uuid):
+    record = LiteratureRecord.get_record(record_uuid)
+    if "Literature" not in record["_collections"]:
+        return
+    authors = record.get("authors", [])
+    updated_authors = []
+    for author in authors:
+        authors_found = AuthorsSearch().disambiguate_author(author).execute().hits
```

Comment:
I think the idea was to use `inspire-matcher` here, as it should make things easier in the future when more requirements are added. Not sure though, might be worth getting other opinions.